ceph-releases/ALL/ubi8/daemon-base/__DOCKERFILE_PREINSTALL__

```diff
@@ -1,17 +1,14 @@
 RUN rm -f /etc/yum.repos.d/ubi.repo
 
-# Editing /etc/redhat-storage-server release file
-RUN echo "Red Hat Ceph Storage Server 5 (Container)" > /etc/redhat-storage-release
-
 EXPOSE 6789 6800 6801 6802 6803 6804 6805 80 5000
 
 # Atomic specific labels
 LABEL version="5"
 
 # Build specific labels
-LABEL com.redhat.component="rhceph-container"
-LABEL name="rhceph"
-LABEL description="Red Hat Ceph Storage 5"
-LABEL summary="Provides the latest Red Hat Ceph Storage 5 on RHEL 8 in a fully featured and supported base image."
-LABEL io.k8s.display-name="Red Hat Ceph Storage 5 on RHEL 8"
-LABEL io.openshift.tags="rhceph ceph"
+LABEL com.redhat.component="ibm-ceph-container"
+LABEL name="ibm-ceph"
+LABEL description="IBM Ceph Storage 5"
```

Comment:
shouldn't it be `IBM Storage Ceph 5`?

Reply:
I'll drop `Storage`, because I think it's just named "IBM Ceph"

Reply:
Or, actually you would know better than me! What are you using in the dashboard?

Reply:
We use IBM Storage Ceph actually. It was the agreed product name according to our designs..